playground/src/app.js

```diff
@@ -24,7 +24,7 @@ function start() {
   registerScreens();
   Navigation.events().onAppLaunched(() => {
     Navigation.setDefaultOptions({
-      _animations: {
+      animations: {
```

Comment:
Lets avoid using custom animations for now, or at least make the animation look decent. :)